ttnn/cpp/ttnn/operations/examples/example_multiple_return/device/example_multiple_return_device_operation.cpp

```diff
@@ -12,10 +12,21 @@ ExampleMultipleReturnDeviceOperation::program_factory_t ExampleMultipleReturnDev
 }
 
 void ExampleMultipleReturnDeviceOperation::validate_on_program_cache_miss(
-    const operation_attributes_t& attributes, const tensor_args_t& tensor_args) {}
+    const operation_attributes_t& attributes, const tensor_args_t& tensor_args) {
+        TT_FATAL(attributes.return_output1 || attributes.return_output2,
+        "At least one output must be returned. return_output1 = {}, return_output2 = {} ",
+        attributes.return_output1,
+        attributes.return_output2);
+
+}
 
 void ExampleMultipleReturnDeviceOperation::validate_on_program_cache_hit(
-    const operation_attributes_t& attributes, const tensor_args_t& tensor_args) {}
+    const operation_attributes_t& attributes, const tensor_args_t& tensor_args) {
```

Comment:
It’s ok to call validate_on_program_cache_miss here

Reply:
Typically, `cache_hit` involves fewer checks, while `cache_miss` performs the checks of `cache_hit` plus additional ones. Wouldn't it make sense for `cache_miss` to call `cache_hit`?

Reply:
Good point, agreed. In this place I just don’t want to duplicate code in between.

Reply:
I will remove duplicate code shortly.

Reply:
```C++
void ExampleMultipleReturnDeviceOperation::validate_on_program_cache_miss(
    const operation_attributes_t& attributes, const tensor_args_t& tensor_args) {
    validate_on_program_cache_hit(attributes, tensor_args);
}
```
I've removed the duplicated code.